CHANGELOG/CHANGELOG-1.20.md

```diff
@@ -1295,7 +1295,7 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
   When we switched to go-runner for building the apiserver,
   controller-manager, and scheduler server components, we no longer
   reference the individual architectures in the image names, specifically
-  in the 'FROM' directive of the server image Dockerfiles.
+  in the 'FROM' directive of the server image Dockerfile.
```

Comment:
I think `Dockerfiles` is right here, plural seem intentional

Reply:
Yeah, corresponding to the before `image names`, we should `Dockerfiles` use here.